src/libraries/System.Formats.Tar/src/Resources/Strings.resx

```diff
@@ -0,0 +1,252 @@
+    <value>The stream does not support writing.</value>
+  </data>
+  <data name="IO_PathNotFound_NoPathName" xml:space="preserve">
+    <value>Could not find a part of the path.</value>
+  </data>
+  <data name="IO_PathNotFound_Path" xml:space="preserve">
+    <value>Could not find a part of the path '{0}'.</value>
+  </data>
+  <data name="IO_PathTooLong" xml:space="preserve">
+    <value>The specified file name or path is too long, or a component of the specified path is too long.</value>
+  </data>
+  <data name="IO_PathTooLong_Path" xml:space="preserve">
+    <value>The path '{0}' is too long, or a component of the specified path is too long.</value>
+  </data>
+  <data name="IO_SetLengthRequiresSeekingAndWriting" xml:space="preserve">
+    <value>SetLength requires a stream that supports seeking and writing.</value>
+  </data>
+  <data name="IO_SharingViolation_File" xml:space="preserve">
+    <value>The process cannot access the file '{0}' because it is being used by another process.</value>
+  </data>
+  <data name="IO_SharingViolation_NoFileName" xml:space="preserve">
+    <value>The process cannot access the file because it is being used by another process.</value>
+  </data>
+  <data name="IO_StreamDisposed" xml:space="preserve">
+    <value>Cannot access a disposed stream.</value>
+  </data>
+  <data name="IO_StreamNotEmpty" xml:space="preserve">
+    <value>The stream is not empty.</value>
+  </data>
+  <data name="PlatformNotSupported_SystemFormatsTar" xml:space="preserve">
+    <value>System.Formats.Tar is not supported on this platform.</value>
+  </data>
+  <data name="SetLengthRequiresSeekingAndWriting" xml:space="preserve">
+    <value>SetLength requires a stream that supports seeking and writing.</value>
+  </data>
+  <data name="TarDuplicateExtendedAttribute" xml:space="preserve">
+    <value>The entry '{0}' has a duplicate extended attribute.</value>
+  </data>
+  <data name="TarUnexpectedMetadataEntry" xml:space="preserve">
+    <value>A metadata entry of type '{0}' was unexpectedly found after a metadata entry of type '{1}'.</value>
+  </data>
+  <data name="TarEntriesInDifferentFormats" xml:space="preserve">
+    <value>The archive contains entries in different formats.</value>
+  </data>
+  <data name="TarEntryBlockOrCharacterExpected" xml:space="preserve">
+    <value>Cannot set the DevMajor or DevMinor fields on an entry that does not represent a block or character device.</value>
+  </data>
+  <data name="TarEntryHardLinkOrSymLinkExpected" xml:space="preserve">
+    <value>Cannot set the LinkName field on an entry that does not represent a hard link or a symbolic link.</value>
+  </data>
+  <data name="TarEntryModeOutOfRange" xml:space="preserve">
+    <value>The mode must be a base 10 number between 0 and 511 (777 in octal).</value>
+  </data>
+  <data name="TarEntryNotARegularFile" xml:space="preserve">
+    <value>Entry type is not a regular file, so it does not support setting the data stream.</value>
```

Comment:
nit: include EntryType in the message? stuff like that is useful for debugging.

Reply:
Included.